adafruit_is31fl3731/__init__.py

```diff
@@ -305,31 +325,67 @@ def pixel_addr(x, y):
         """Calulate the offset into the device array for x,y pixel"""
         return x + y * 16
 
-    # pylint: disable-msg=too-many-arguments
-    def pixel(self, x, y, color=None, blink=None, frame=None):
+    # pylint: disable-msg=too-many-arguments, too-many-branches
+    def pixel(
+        self,
+        x: int,
+        y: int,
+        color: int = 255,
```

Comment:
I think `color` should remain with `None` as it's default which would also make it `Optional[]`

`frame` has some logic checking for it's value to be `None` so I think it's default value should go back to `None` instead of `0` as well.